### PR TITLE
add libblas symlinks

### LINKED ISSUE #23: Provide libblas.so and liblapack.so
Any reason why https://github.com/conda-forge/openblas-feedstock/blob/master/recipe/build.sh#L46-L49 is commented?

### PR DESCRIPTION
**Pull request by isuruf**
Fixes #23

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ unset CFLAGS
 # Depending on our platform, shared libraries end with either .so or .dylib
 if [[ `uname` == 'Darwin' ]]; then
      export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
-#    DYLIB_EXT=dylib
+     DYLIB_EXT=dylib
 else
      export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
-#    DYLIB_EXT=so
+     DYLIB_EXT=so
```
Comment (jakirkham) [inline recipe/build.sh]: So I don't think we need these any more. One can use [`SHLIB_EXT`]( https://conda.io/docs/building/environment-vars.html#environment-variables-set-during-the-build-process ). Though that includes the `.`. Correct @jschueller?

**Comment by jakirkham** [comment]
Would also need a build number bump.

**Comment by jakirkham** [comment]
Personally I'm ok with doing something like this. I think I had held off on doing it as we thought people might want to mix BLASes initially, but it has become clear after the brief disruption of having NumPy use OpenBLAS and SciPy use MKL that this is really not feasible as one will get crashes. Not sure if there are any other downsides to this.

**Comment by isuruf** [comment]
@jakirkham, I opened an alternative proposal https://github.com/conda-forge/blas-feedstock/pull/7. Problem with that is that there is a version bump and all conda-forge packages pin the `blas` package with the exact version.

**Comment by jakirkham** [comment]
Changes like these seem to require knowledge of the specific package. So it would be better for these details to live in the packages that implement them IMHO.

FWIW there has been discussion at OpenBLAS about including libraries like these as output of the build. ( https://github.com/xianyi/OpenBLAS/issues/296 ) So if we have to change it in the future, it would be better if we just change the `openblas` package and not the `blas` package.

**Comment by jakirkham** [comment]
LGTM. Other thoughts @conda-forge/openblas?

**Comment by isuruf** [comment]
ping @conda-forge/openblas

**Comment by ocefpaf** [comment]
Thanks @isuruf!

**Comment by isuruf** [comment]
Thanks for the reviews and the merge

**Pull request merged**